modules/core/src/main/java/org/mycontroller/standalone/message/McMessage.java

```diff
@@ -54,6 +54,7 @@
     @NonNull
     private Integer gatewayId;
     private String nodeEui;
+    private String name;
```

Comment:
@jkandasa I'm not sure I understand, this actually work with MySensors and MyController. I actually using it. 

RFLink method used `sensorId` as `name` I can't do the same. 
Because `sensorId` is used by hue API and the name is used on dashboard :
<img width="300" alt="dashboard_lights_with_name" src="https://cloud.githubusercontent.com/assets/686174/21836178/0d62cb00-d791-11e6-9996-07fa1b9f23de.png">

And not having the name or if I had to add each light name. Will not really fun for end user (And I only have seven light). 
Here no `name` but `sensorId` is set:
<img width="280" alt="dashboard_lights_no_name" src="https://cloud.githubusercontent.com/assets/686174/21836177/0d2ea456-d791-11e6-86dd-82d1e8827daf.png">